web/packages/teleterm/src/mainProcess/createAgentConfigFile.ts

```diff
@@ -50,16 +48,23 @@ export async function createAgentConfigFile(
     }
   }
 
+  const labels = Object.entries({
+    // TODO(ravicious): Move this to a JavaScript version of constants.go.
+    'teleport.dev/connect-my-computer/owner': args.username,
```

Comment:
As I wrote in the description, this is just temporary and the next commit will move this and another label to a better location. I just didn't want to spend time on thinking about it upfront since coming up with that new location wasn't needed for this particular PR.